### ISSUE TITLE
Project Feedback!

### ISSUE THREAD

**Issue by codepathreview** [fhuang26/SimpleTweets_fh#1]
@fhuang26 : The requirement *User is displayed the username, name, and body for each tweet* is incomplete. Pls complete and update status here

**Comment by fhuang26** [comment]
Are we expected to display both name and screen_name in "user" of tweet
json? In studio debugger, I browsed tweet Json object and didn't find name
or title in a few tweets. Could you help remind me of what is missing in
the version of project 3 I checked in?  Thank you, Felix

**Comment by codepathreview** [comment]
@fhuang26 : yes the requirement is to have both username and name as part of each tweet in the list.  
/cc @codepathreview

**Comment by fhuang26** [comment]
Both name and screen_name of "user" in tweet Json object are displayed in
every tweet in the list. I re-submitted the revised project 3 as follows.
  git repo :  https://github.com/fhuang26/SimpleTweetsFH
  gif :  https://i.imgur.com/etU2q9H.gif
gif is also attached.

Best regards,
Felix



On Mon, Oct 2, 2017 at 4:07 PM, CodePath Reviewers <notifications@github.com
